src/main/java/org/apache/mesos/hdfs/Scheduler.java

```diff
@@ -16,17 +16,12 @@
 import org.apache.mesos.hdfs.state.PersistentState;
 import org.apache.mesos.hdfs.util.HDFSConstants;
 
+import java.io.IOException;
+import java.net.InetAddress;
+import java.net.Socket;
 import java.sql.Timestamp;
-import java.util.ArrayList;
-import java.util.Arrays;
+import java.util.*;
```

Comment:
I usually prefer not to use 'import *', so that we're clear about specifically what we're importing and there's less of a chance of an internal classname conflicting with an (unused) imported class.